requirements_test_min.txt

```diff
@@ -5,3 +5,4 @@ typing-extensions~=4.2
 pytest~=7.1
 pytest-benchmark~=3.4
 pytest-timeout~=2.1
+requests
```

Comment:
I don't think we need request installed during our tests ?

```suggestion
```

Reply:
We are using a `infered` way to get the name of the method

If the library is not installed the unittest fails since that the check is not detected

My first approach was without infered (not library needed):
 - https://github.com/PyCQA/pylint/pull/6885

Check the following discussion about this:
 - https://github.com/PyCQA/pylint/pull/6780#discussion_r889506944

and last review where he confirms it is needed:
 - https://github.com/PyCQA/pylint/pull/6780#pullrequestreview-1001058586

Reply:
Hmm strange, it worked for me locally without installing request. Would you mind applying the suggestion and see how the continuous integration react ?

Reply:
Try uninstalling it explicilty
`pip uninstall requests`

Even, try to use the code added in rst doc file

```python
from astroid import extract_node
node = extract_node('from requests import get;get()')
print(next(node.func.infer()).qname())
```

In fact, it is the way like infered works:
 - ![Screen Shot 2022-06-10 at 15 28 37](https://user-images.githubusercontent.com/6644187/173145630-514cdfa6-5233-411a-8056-64bcb9abec48.png)

> Would you mind applying the suggestion and see how the continuous integration react ?

I did, the previous commit has that proof and the CI failed in the unittest of this check

Reply:
I think adding `requests` to our test dependencies is not unreasonable. Many environments will already have it since the package is used so often in the Python ecosystem.

I'll leave final decision to @Pierre-Sassoulas but I would be okay with adding it here.

We could also add it as an additional test dependencies and only run the test when it is installed.